site/oc/pages.clj

```diff
@@ -873,8 +873,6 @@
           [:script {:type "text/javascript" :src "/lib/autotrack/autotrack.js"}]
           [:script {:type "text/javascript" :src "/lib/autotrack/google-analytics.js"}]
           (google-analytics-init)
-          [:script {:type "text/javascript" :src "/lib/fullstory.js"}]
```

Comment:
The script tag seems missing below

Reply:
@thepug do you see an error? You shouldn't, i removed fullstory.js only from the local dev app-shell, that was using it directly. In `prod-app-shell` we use the version inside oc-assets.js that is compiled using `compile_assets.sh`.